src/screens/moviesScreen/MovieScreen.tsx

```diff
@@ -1,12 +1,80 @@
+import {useEffect, useState} from 'react';
+import {ScrollView} from 'react-native';
+
+import Carousel from '@components/carousel';
+import HorizontalMovieScroll from '@components/horizontalMovieScroll';
 import BaseContainer from '@components/baseContainer';
-import Movies from '@components/movies';
+import Loader from '@components/loader';
+
+import {discover} from '@network/apiUrls';
+import {fetchData} from '@network/apiMethods';
+
+import {useThemeColors} from '@theme/themes';
+import {styles} from './styles';
```

Comment:
Import ordering should be as follows with proper line breaks -

import {useEffect, useState} from 'react';
import {ScrollView} from 'react-native';

import Carousel from '@components/carousel';
import HorizontalMovieScroll from '@components/horizontalMovieScroll';
import BaseContainer from '@components/baseContainer';
import Loader from '@components/loader';
import {discover} from '@network/apiUrls';
import {fetchData} from '@network/apiMethods';
import {useThemeColors} from '@theme/themes';

import {styles} from './styles';